packages/aws-cdk-lib/aws-ecs/README.md

```diff
@@ -340,6 +340,17 @@ cluster.addCapacity('ASGEncryptedSNS', {
 });
 ```
 
+### Container Insights
+
+On a cluster, CloudWatch Container Insights can be enabled by setting the `containerInsightsV2` property. [Container Insights](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/cloudwatch-container-insights.html) 
+can be disabled, enabled, or enhanced. Omitting the property will not default any setting, and the cluster will inherit its Container Insights configuration from the account level Container Insights settings. 
```

Comment:
Again, "not default any settings" makes it sound nondeterministic, but I would just forego the default value explanation here. The "can be enabled" at the start of the chapter implies that there is a disabled default strongly enough for me, and the JSDoc for `containerInsightsV2` clears up the rest. 

```suggestion
can be disabled, enabled, or enhanced.
```

Reply:
Sounds good.